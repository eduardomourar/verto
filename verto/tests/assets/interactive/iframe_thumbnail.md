{interactive name="binary-cards" type="iframe" thumbnail="binarycards.png"}
