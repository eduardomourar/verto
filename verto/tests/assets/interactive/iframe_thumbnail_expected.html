<iframe src="{% url 'interactive' name='binary-cards' %}">
<p>Your browser does not support iframes.</p>
</iframe>
